google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudStackFrame.java

```diff
@@ -148,7 +148,7 @@ public MyValue(@NotNull Variable variable, @NotNull List<Variable> variableTable
       this.variable = variable.getVarTableIndex() != null ? variableTable
           .get(variable.getVarTableIndex().intValue())
           : variable;
-      members = variable.getMembers();
+      members = this.variable.getMembers();
```

Comment:
My understanding is that the server compresses (or simply serializes) the variables table in different ways and, somewhere along the way, they changed the way they do it.

We're getting `arguments` and `locals` from CDB. Each of these instances contains a `varTableIndex` attribute, which is a pointer to an index of a `variableTable`, kept alongside these `variables` and `locals`.

The existing code seems to suggest that this variable information used to be stored inside the `arguments` and `locals` instances, and now it's stored alongside them. Hope this is clear!